tests/BasicsTest.php

```diff
@@ -241,39 +241,6 @@ public function testSetLongUserAgent() {
         $this->assertEquals(strlen($md->getUserAgent()), 500);
     }
 
-    public function testSetDetectionType()
```

Comment:
Why was this removed? At the moment, just looking at the code I don't remember the actual functionality and purpose.